src/main/java/com/dev/cinema/dao/hibernate/MovieDaoImpl.java

```diff
@@ -0,0 +1,55 @@
+package com.dev.cinema.dao.hibernate;
+
+import com.dev.cinema.dao.MovieDao;
+import com.dev.cinema.exceptions.DataProcessingException;
+import com.dev.cinema.model.Movie;
+import com.dev.cinema.util.HibernateUtil;
+import org.apache.log4j.Logger;
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+
+import javax.persistence.criteria.CriteriaBuilder;
+import javax.persistence.criteria.CriteriaQuery;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * @author Sergey Klunniy
+ */
+public class MovieDaoImpl implements MovieDao {
+
+    private static final Logger logger = Logger.getLogger(MovieDaoImpl.class);
```

Comment:
```suggestion
    private static final Logger LOGGER = Logger.getLogger(MovieDaoImpl.class);
```
який неймінг у констант?

Reply:
константи пишуться великими літерами